packages/now-build-utils/src/detect-builders.ts

```diff
@@ -33,8 +34,9 @@ function getBuilders({ tag }: Options = {}): Map<string, Builder> {
 
 // Must be a function to ensure that the returned
 // object won't be a reference
-function getApiBuilders({ tag }: Options = {}): Builder[] {
+function getApiBuilders({ tag }: Pick<Options, 'tag'> = {}): Builder[] {
```

Comment:
What is this `Pick` stuff for?

Reply:
It's there to make it more obvious that just this property or properties are used here. e.g. since sometimes the function gets exported and is used for `detectRoutes` as well, but it doesn't have all the options there, so it will just use what it has.